src/rust/scheduler/scheduler.rs

```diff
@@ -164,16 +164,17 @@ impl Scheduler {
             pages.push(WakerPageRef::default());
         }
         let (page, subpage_ix): (&WakerPageRef, usize) = {
-            let (pages_ix, subpage_ix) = self.get_page_indexes(pin_slab_index);
+            let (pages_ix, subpage_ix) = self.get_page_and_offset_indexes(pin_slab_index);
             (&pages[pages_ix], subpage_ix)
         };
         page.initialize(subpage_ix);
         Some(TaskHandle::new(task_id, pin_slab_index, page.clone()))
     }
 
-    /// Computes the page and page offset of a given task based on its total offset.
```

Comment:
Please, do not remove existing comments.

Reply:
I was hoping to make it clear in the function name so that we don't need the comment. Is it still ambiguous?

Reply:
Well, I have already expressed my opinion on this many times... I am not arguing against your renaming, but I am not ok with removing comments. I think that this is a middle ground, right?

Reply:
We should consider such opinions case by case and cannot treat them as thumb rules. But, I will keep this comment because it does add context about the total offset.

Reply:
I think we should leave comments for every function because we may want to generate rust docs in the future. In fact, I would advocate that we eventually document every input and output to every function. Sometimes I find myself searching the code for what the function actually outputs and it's really annoying.

Reply:
Ok, I reverted the comment. We can discuss offline about more on this, if needed.